dotnetv3/Glue/Actions/HelloGlue.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace GlueActions;
```

Comment:
I see the HelloGlue, but I don't see any snippets or it in the metadata.

Reply:
Thanks for catching that. Added Hello Glue to the metadata.